IDEOorg/steps-bot
# #164125599 As a developer, I should have faith that in the event the application crashes, the users should not suffer an indefinite downtime

#### What does this PR do?

Adds a package manager library to the api. The library itself is called PM2.

#### Description of Task to be completed?

- Adds PM2 library to the requirements of the bot application.
- The library will restart the application after 3s in the event the application crashes

#### How should this be manually tested?

- clone the repo
- checkout to the branch `ch-integrate-pm2-with-bot-164125599`
- install requirements with `npm install`
- Run the application in production by executing `npm run start:prod`
- Any attempts to crash the application should result in the application restarting after 3s.

#### What are the relevant pivotal tracker stories?

[#164125599](https://www.pivotaltracker.com/story/show/#164125599)

#### Screenshots

- N/A
